lenskit-core/src/main/java/org/grouplens/lenskit/data/sql/JDBCRatingDAO.java

```diff
@@ -136,8 +136,14 @@ public JDBCRatingDAO(Connection dbc, SQLStatementFactory sfac, boolean close) {
         userStatement = new CachedPreparedStatement(dbc, statementFactory.prepareUsers());
         itemStatement = new CachedPreparedStatement(dbc, statementFactory.prepareItems());
         for (SortOrder order : SortOrder.values()) {
-            eventStatements.put(order, new CachedPreparedStatement(dbc,
-                                                                   statementFactory.prepareEvents(order)));
+            switch (order) {
+            case ASCENDING:
```

Comment:
oof, `break` is confusing in this case. It breaks out of the `switch`, which, since the `switch` is the last element of the `for`, continues the `for`.

This particular code is a temporary stop-gap while both DAO interfaces are in LensKit. As soon as the old DAOs are gone, this code will be gone.